README.md

```diff
@@ -156,9 +177,6 @@ Future releases targets include:
 
 | Component                           | Version    | Target Date  |
 | ----------------------------------- | ---------- | ------------ |
-| W3C Correlation Context Propagation | Beta v1    | March 16 2020|
-| Support for Tags/Baggage            | Beta v1    | March 16 2020|
-| gRPC Integrations                   | Beta v1    | March 16 2020|
-| OpenTelemetry Collector Exporter    | Beta v1    | March 16 2020|
-| OpenCensus Bridge                   | Beta v1    | March 16 2020|
-| Metrics SDK (Complete)              | Beta v1    | March 16 2020|
+| Support for Tags/Baggage            | Beta v1    | March 30 2020|
+| gRPC Integrations                   | Beta v1    | March 30 2020|
+| OpenCensus Bridge                   | Beta v1    | March 30 2020|
```

Comment:
```suggestion
| Stable API for metrics and tracing | Beta v2 | March 31 2020 |
| Support for Tags/Baggage           | Beta v2 | March 31 2020 |
| gRPC Integration                   | Beta v2 | March 31 2020 |

```

I don't think we'll have the OC bridge for the beta, but I think we should note that this is when the APIs are supposed to stop changing.

Feel free to accept or reject the release name and date changes.